### PR TITLE
Add Indosum dataset

### PR DESCRIPTION
**Pull request by prasastoadi**

### CODE REVIEW

File: datasets/indosum/indosum.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepaths, split):
+        """ Yields examples. """
+
+        logging.info("⏳ Generating examples from = %s", filepaths)
+        dataset = load_dataset("json", data_files=filepaths)
```
Comment (lhoestq) [inline datasets/indosum/indosum.py:152]: Can you load the files using the `json` module here instead ?

Indeed the dataset scripts are not supposed to write files on disk and `load_dataset` does create cache files (arrow files).

**Review by lhoestq — COMMENTED** [review]
This is really cool thanks :) 

I left a few comments

**Comment by lhoestq** [comment]
Hi @prasastoadi have you had a chance to take a look at my suggestions ?

Feel free to ping ;e if you have questions or when you're ready for a review

**Comment by albertvillanova** [comment]
Thanks for your contribution, @prasastoadi. Are you still interested in adding this dataset?

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

We would suggest you create this dataset there. Please, feel free to tell us if you need some help.

**Pull request closed without merging**